zwave_js_server/model/node.py

```diff
@@ -310,8 +310,36 @@ def receive_event(self, event: Event) -> None:
 
         self.emit(event.type, event.data)
 
+    async def _async_send_command(
+        self,
+        cmd: str,
+        require_schema: Optional[int] = None,
+        wait_for_result: bool = None,
```

Comment:
Why put `None` as default? We don't seem to care about it in the method body. It could be `False`.

Reply:
That’s a good point, when I had written this originally I expected to use None but it’s not required. Will update it later

Reply:
I recall now why we needed None, and I'm glad you mentioned it because I didn't handle it correctly. The idea was that the caller could force a behavior (wait or not wait) with a bool whereas `None` would be based on node status. I'll push a commit that implements this properly

Reply:
```suggestion
        wait_for_result: Optional[bool] = None,
```